api/app/signals/apps/api/v1/filters/signal.py

```diff
@@ -232,6 +233,12 @@ def assigned_user_email_filter(self, queryset, name, value):
         else:
             return queryset.filter(user_assignment__user__email__iexact=value)
 
+    def assigned_routing_filter(self, queryset, name, value):
```

Comment:
Why `assigned_routing_filter` and not just `routing_department_code_filter`?

Reply:
done

Reply:
done